### ISSUE TITLE
Homey version 2.0

### ISSUE THREAD

**Issue by PeterBox** [kerkenit/nl.marcovantklooster.pioneer#8]
App is not working  with Homey version 2.0

**Comment by PeoplewarePPC** [comment]
Yep.. unfortunately I found out when I updated to 2.0 :( 
Could you please update the app? I will buy you a beer.

**Comment by dENNtoet** [comment]
Yep. Unfortunately I found out when I did do the update to 2.0 :(
Could you please update the app? Will buy you a beer.
